mod_reforged/hooks/skills/effects/taunted_effect.nut

```diff
@@ -19,11 +21,50 @@
 			});
 		}
 
+		if (this.getMeleeDefenseModifier() != 0)
+		{
+			ret.push({
+				id = 13,
+				type = "text",
+				icon = "ui/icons/melee_defense.png",
+				text = ::MSU.Text.colorizeValue(this.getMeleeDefenseModifier()) + " Melee Defense"
```

Comment:
Please add hyperlinks for the concepts such as melee defense and ranged defense.